presto-docs/src/main/sphinx/release/release-0.211.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.211
+=============
+
+General Changes
+---------------
+
+* Fix missing final query plan in ``QueryCompletedEvent``. Statistics and cost estimates
+  are removed from the plan text because they may not be available during event generation.
+* Update the default value of the `http-server.https.excluded-cipher` config
+  property to match Jetty's default excluded cipher suites. Specifically, all
+  ciphers without forward secrecy, and all ciphers that use the RSA key exchange
+  algorithm are excluded by default. Consequently, TLS 1.0 or TLS 1.1 are no
+  longer supported by default. `http-server.https.excluded-cipher` config property
+  can be set to empty string to restore the old behavior.
+* Allow running on a JVM from any vendor that meets the functional requirements.
+* Add :func:`ST_GeomFromBinary` and :func:`ST_AsBinary` functions that converts
+  geometries to and from Well-Known Binary format.
+* Remove the ``verbose_stats`` session property. Rename the ``task.verbose-stats``
```

Comment:
How about merging the sentences with "and". Otherwise, it seems like they should be separate release note items.